HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MapObjectEntity.cs

```diff
@@ -24,6 +30,8 @@ public SolidColorBrush getColor()
         }
 
         public MapObjectType MapObjectType { get => _mapObjectType; set => _mapObjectType = value; }
+        public MapObjectTypes GetMapObjectTypes { get => _mapObjectType.GetMapObjectTypes; }
```

Comment:
I think that having both of these properties in unnecessary, they expose the same data. I would probably only keep the second, but rename it.

Reply:
fixed